### PR TITLE
remove mention of access_token in doc comment

### PR DESCRIPTION
**Pull request by ijsnow**
## Description of the change

When logging errors to stderr is enabled, we print access tokens which is a security concern.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance
- [ ] New release

## Related issues

> Shortcut stories and GitHub issues (delete irrelevant)

- Fix [SC-]
- Fix #1

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

File: transport.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func rollbarError(logger ClientLogger, format string, args ...interface{}) {
 }
 
 func writePayloadToStderr(logger ClientLogger, payload map[string]interface{}) {
+	if _, ok := payload["access_token"]; ok {
```
Comment (pawelsz-rb) [inline transport.go:86]: what sets "access_token" in the payload? why do we need to do it?

**Review by pawelsz-rb — COMMENTED** [review]
comment

File: transport.go (hunk 1 above)
Comment (pawelsz-rb) [reply transport.go]: yeah, that's why I suggested @rollbar-bborsits to get back to the customer and ask them to upgrade. They may be using a really old version.

Comment (rollbar-bborsits) [reply transport.go]: I asked the customer, but they have yet to reply. We had some internal miscommunication hence this PR.

Comment (jmarronm) [reply transport.go]: Any update from the customer about this stale PR @rollbar-bborsits ?

It seems not valid in general based on the previous comments

Comment (rollbar-bborsits) [reply transport.go]: @jmarronm Yes, you're right. This is not a valid use-case, so we can close this PR.

**Pull request closed without merging**